ros_ws/src/teleoperation/include/joystick_map_ps3.h

```diff
@@ -0,0 +1,33 @@
+#include "joystick_map.h"
+
+class JoystickMapPs3 : public JoystickMap
+{
+    public:
+    JoystickMapPs3()
+    {
+    }
+
+    ~JoystickMapPs3()
+    {
+    }
+
+    float getSteeringAxis(const sensor_msgs::Joy::ConstPtr& joystick)
```

Comment:
**MAY**
Add the virtual keyword to make it more clear, that this is a virtual member function.